### ISSUE TITLE
IPR039796

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#2217]

**Comment by ValWood** [comment]
should this be
protein maturation by [4Fe-4S] cluster transfer

rather than

GO:0016226 | iron-sulfur cluster assembly

? @Antonialock

**Comment by ValWood** [comment]
Actually the ISO is to 

protein maturation by [4Fe-4S] cluster transfer | ISO with S000001164

but SGD have GO:0016226 | iron-sulfur cluster assembly?

Assigning to @Antonialock  to add solution

**Comment by Antonialock** [comment]
The paper I ISSd to seems to suggest it is [4Fe-4S] cluster transfer

<img width="691" alt="screenshot 2018-12-10 at 21 47 48" src="https://user-images.githubusercontent.com/12727506/49763737-6b138b80-fcc5-11e8-8cca-1bc675233701.png">

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4523923/

**Comment by Antonialock** [comment]
(IPR039796 is in cia2)

**Comment by Antonialock** [comment]
Val I added you back in, I'm not sure what you mean by 'Antonia to add solution'

**Comment by Antonialock** [comment]
unless you mean 'suggest solution' in which case I suggest that IPR039796 should map to the maturation term :-)

**Comment by ValWood** [comment]
That's OK, you answered the question !

**Comment by asangrador** [comment]
Agreed. I have changed the term to 'protein maturation by [4Fe-4S] cluster transfer'. There must be some confusion with these terms because UniProt and SGD have assigned  ' iron-sulfur cluster assembly'  to some of these proteins (Q9H5X1 (CIA2A_HUMAN), P38829 (YHS2_YEAST).

**Comment by ValWood** [comment]
Yup, @Antonialock  when you went through these did you report the fixes required?
If not could you open an "annotation review" ticket and tag people to look at these?

**Comment by ValWood** [comment]
cia2 | CIA machinery protein Cia2 (predicted) |   | iron-sulfur cluster assembly |   | GO_Central | Schizosaccharomyces pombe | IBA | PANTHER:PTN000272123SGD:S000001164 more... | uncharacterized pthr12377 | protein |   | PMID:21873635 | 20170228

should be  'protein maturation by [4Fe-4S] cluster transfer'. it's post-assembly @Antonialock  correct?

**Comment by ValWood** [comment]
Oddly, when I look in AmiGO for  other annotations to  "iron-sulfur cluster assembly" for  PANTHER:PTN000272123 I do not find any annotations top this term that are members of 

PANTHER:PTN000272123. Even the one above. Is this a bug?

@pgaudet

**Comment by ValWood** [comment]
@Antonialock  should the SGD annotation be rrrevised. That is the annotation that the PAINT [iron-sulfur cluster assembly](https://www.yeastgenome.org/go/GO:0016226) is coming from?

But   what is the name for the entire process? (cluster assembly/ transfer and insertion?)
You would want a term to link these annotations together.

Now I think about it "protein maturation by [4Fe-4S] cluster transfer" seems to be a MF in BP?
